connector/ldap/ldap.go

```diff
@@ -50,6 +50,11 @@ import (
 //         groupAttr: member
 //         nameAttr: name
 //
+//         #Used when the filter member:1.2.840.113556.1.4.1941: timesout
```

Comment:
About how many groups does it take for this query to timeout?

Reply:
I don't really know but I tried this and it timed out after one minute. I find it unacceptable to have queries that last longer than 20 seconds (Since it's just a login).
With this implementation, the duration of login reduced in just a couple of seconds